apps/uriparser.cpp

```diff
@@ -223,11 +224,13 @@ void UriParser::Parse(const string& strUrl, DefaultExpect exp)
 
 #ifdef TEST
 
-
 using namespace std;
 
 int main( int argc, char** argv )
 {
+    if (argc < 2) {
```

Comment:
Please keep up to the current convention (brace opened in the new line).

Reply:
Done. Also fixed spacing.